.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of the problem you are facing and what you expected to happen instead.
+
+**Ansible error**
+Copy and paste the Ansible error message while running this playbook (inside below code fences):
+
+```shell
+
+```
+
+**Ansible context**
+Paste the full output of `ansible --version`
```

Comment:
I think we could ask also for 
`ansible-config dump --only-changed` and `ansible-inventory -i inventory_file --graph`

Reply:
and target OS is always useful and shouldn't bother people too much